### PR TITLE
Added a number of new and very nerdy names for the AI

### PR DESCRIPTION
**Pull request by Moomoobeef**
## About the PR
I added as many nerdy new AI names as I could think of. Not *all* of them are a reference to something, but most are. And I explained some of the more obscure ones in comments in the file.

## Why / Balance
this originally started purely because I wanted to add CompuServe after I noticed we already had AOL in the list. So I did... and then I kept going.

Big nerds (see: ss14 players) may appreciate MORE nerdy references

## Requirements
- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**
:cl:
- add: Added more names to the pool of names the AI can have.

### CODE REVIEW

**Comment by lzk228** [comment]
nerdy

File: Resources/Prototypes/Datasets/Names/ai.yml [hunk 1]
```diff
@@ -14,26 +17,35 @@
   - Alpha 7
   - Alpha 8
   - Alpha 9
+  - K6-II #AMDs first PC processor to actually be really succesful [citation needed]
```
Comment (slarticodefast) [inline Resources/Prototypes/Datasets/Names/ai.yml:20]: put this one into the list in alphabetical order

File: Resources/Prototypes/Datasets/Names/ai.yml [hunk 2]
```diff
@@ -89,3 +111,4 @@
   - Z-2
   - Z-3
   - Zed
+  - Zuse Z3 #influential early computer important in history for being the first fully automatic digital computer
```
Comment (slarticodefast) [inline Resources/Prototypes/Datasets/Names/ai.yml:114]: We already have Z-3

**Comment by slarticodefast** [comment]
Also I'm not nerdy enough to understand all of these, lol

**Comment by superjj18** [comment]
I'm hoping when AI loadouts come out, they will include an "AI Name" field

**Comment by Moomoobeef** [comment]
> I'm hoping when AI loadouts come out, they will include an "AI Name" field

That would be ideal, but until then we must persevere

**Comment by slarticodefast** [comment]
PJB said to drop the Microsoft/Intel/AMD products so we don't violate any trademarks.

**Comment by Psychpsyo** [comment]
I don't think that's how trademarks work.

**Comment by Moomoobeef** [comment]
> PJB said to drop the Microsoft/Intel/AMD products so we don't violate any trademarks.

For some reason I thought I had already made a commit to fix this and was just waiting on re-review. Guess I didn't. I will do that now.

**Comment by Moomoobeef** [comment]
> I don't think that's how trademarks work. (unless we want into the business of selling processors)

you might be right but I'm not a lawyer, so it's probably best to be on the safe side.

**Comment by slarticodefast** [comment]
Thanks for your contribution!

**Pull request merged**